tests/github-action-helper.sh

```diff
@@ -6,11 +6,27 @@ set -xeEo pipefail
 # VARIABLES #
 #############
 : "${FUNCTION:=${1}}"
-: "${BLOCK:=$(sudo lsblk --paths | awk '/14G/ || /75G/ {print $1}' | head -1)}"
+
+function find_extra_block_dev() {
```

Comment:
This is the same function from the rook repo, right? Or any changes? How about adding a comment here for where the rook function is found in case future bug fixes are needed